src/test/java/com/github/davidmoten/rtree2/geometry/PolygonTest.java

```diff
@@ -0,0 +1,93 @@
+package com.github.davidmoten.rtree2.geometry;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public final class PolygonTest {
```

Comment:
Do these tests give us a full coverage of the PolygonDouble class?

Reply:
Class, Method and Line coverage now all 100%